file-system.ts

```diff
@@ -7,7 +6,9 @@ import path = require("path");
 import util = require("util");
 import rimraf = require("rimraf");
 import minimatch = require("minimatch");
+import decorators = require("./decorators");
 
+@decorators.register("fs")
```

Comment:
From a pure API perspective I think it will be better to move the decorators into the injector file itself. It will look like:
```ts
@injector.register("fs")
```